package.json

```diff
@@ -0,0 +1,35 @@
+{
+  "name": "weather-app",
+  "version": "1.0.0",
+  "description": "This is a weather app to test the knowledge of asynchronous functions in Javascript.",
+  "main": "index.js",
+  "scripts": {
+    "dev": "webpack --mode development",
+    "build": "webpack --mode production",
+    "start": "webpack-dev-server --mode development --open"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/Tripple-A/Weather-App.git"
+  },
+  "author": "Biodun",
+  "license": "ISC",
+  "bugs": {
+    "url": "https://github.com/Tripple-A/Weather-App/issues"
+  },
+  "homepage": "https://github.com/Tripple-A/Weather-App#readme",
+  "devDependencies": {
+    "css-loader": "^3.4.2",
+    "eslint": "^6.8.0",
+    "eslint-config-airbnb-base": "^14.0.0",
+    "eslint-plugin-import": "^2.20.0",
+    "file-loader": "^5.0.2",
+    "style-loader": "^1.1.3",
+    "webpack": "^4.41.5",
+    "webpack-cli": "^3.3.10"
+  },
+  "dependencies": {
+    "giphy-api": "^2.0.1",
+    "webpack-dev-server": "^3.10.1"
```

Comment:
This should be in `devDependencies`, as specified [here](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/javascript/weather_app/devdep_requirement.png) [`REQUIRED`]